<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify_data</class>
 <widget class="QDialog" name="Modify_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plis - Modify data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modify the selected dataset</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Resources.qrc">:/img/img/shapes.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Please, select the shape you want for the dataset.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="shape_box">
           <item>
            <property name="text">
             <string>ssDisc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssSquare</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssDiamond</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssStar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssCross</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssPlus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssCircle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssTriangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssTriangleInverted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssCrossSquare</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssPlusSquare</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssCrossCircle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssPlusCircle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ssPeace</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Please, select the color you want of the dataset.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="color_box">
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grey</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darkmagenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darkblue</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Please, select the color you want of the curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="color_curve_box">
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grey</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darkmagenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darkblue</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="2">
            <widget class="QSpinBox" name="size_data"/>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="size_curve"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Size of datapoints:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Size of fitted curve:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
